aws/cloudformation/geocoder.yml

```diff
@@ -0,0 +1,180 @@
+# This template is a modified version of the template generated by the ECS console cluster creation wizard.
+# It creates a fargate service which runs https://github.com/apilayer/freegeoip/, and an ELB which fronts that service.
+
+AWSTemplateFormatVersion: '2010-09-09'
+Description: AWS CloudFormation template to create an ECS Fargate stack for running https://github.com/apilayer/freegeoip/.
+Parameters:
+  EcsClusterName:
+    Type: String
+    Description: ECS Cluster Name
+    Default: geocoder-freegeoip
+  EcsPort:
+    Type: String
+    Description: Optional - Security Group port to open on ECS instances - defaults to port 80
+    Default: '80'
+  ElbPort:
+    Type: String
+    Description: Optional - Security Group port to open on ELB - port 80 will be open by default
+    Default: '80'
+  SourceCidr:
+    Type: String
+    Description: Optional - CIDR/IP range for EcsPort and ElbPort - defaults to 0.0.0.0/0
+    Default: 0.0.0.0/0
+  TargetType:
+    Type: String
+    Description: Optional - the ALB target group target type, defaults to ip
+    Default: 'ip'
+Resources:
+  EcsSecurityGroup:
+    Type: AWS::EC2::SecurityGroup
+    Properties:
+      GroupDescription: ECS Allowed Ports
+      VpcId: !ImportValue VPC
+      SecurityGroupIngress: 
+        -
+          IpProtocol: 'tcp'
+          FromPort: !Ref 'EcsPort'
+          ToPort: !Ref 'EcsPort'
+          CidrIp: !Ref 'SourceCidr'
+        -
+          IpProtocol: 'tcp'
+          FromPort: '1'
+          ToPort: '65535'
+          SourceSecurityGroupId: !Ref 'AlbSecurityGroup'
+        -
+          IpProtocol: 'tcp'
+          FromPort: !Ref 'EcsPort'
+          ToPort: !Ref 'EcsPort'
+          CidrIp: !Ref 'SourceCidr'
+      Tags:
+        -
+          Key: 'Name'
+          Value: !Join [' ', ['ECS', !Ref 'EcsClusterName', '-', 'ECS SecurityGroup']]
+        -
+          Key: 'Description'
+          Value: !Join [' ', ['Created for ECS cluster', !Ref 'EcsClusterName']]
+  AlbSecurityGroup:
+    Type: AWS::EC2::SecurityGroup
+    Properties:
+      GroupDescription: ELB Allowed Ports
+      VpcId: !ImportValue VPC
+      SecurityGroupIngress:
+        - IpProtocol: 'tcp'
+          FromPort: !Ref 'ElbPort'
+          ToPort: !Ref 'ElbPort'
+          CidrIp: !Ref 'SourceCidr'
+      Tags:
+        -
+          Key: 'Name'
+          Value: !Join [' ', ['ECS', !Ref 'EcsClusterName', '-', 'ALB SecurityGroup']]
+        -
+          Key: 'Description'
+          Value: !Join [' ', ['Created for ECS cluster', !Ref 'EcsClusterName']]
+  DefaultTargetGroup:
+    Type: AWS::ElasticLoadBalancingV2::TargetGroup
+    Properties:
```

Comment:
This could also use a specific [`Name`](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-elasticloadbalancingv2-targetgroup.html#cfn-elasticloadbalancingv2-targetgroup-name) Property for better readability.

Reply:
Will update.